src/axom/sidre/tests/spio/spio_parallel.hpp

```diff
@@ -997,13 +997,10 @@ TEST(spio_parallel, sidre_simple_blueprint_example)
   MPI_Barrier(MPI_COMM_WORLD);
 
   // rank 0 adds the bp index to the root file
```

Comment:
Does `writeBlueprintIndexToRootFile()` make sure that rank 0 is the only one that writes?  If the function makes that check, please briefly say so in the comment.  Otherwise, the comment should be removed since the check for rank 0 was removed.